### DISCUSSION TITLE
Would people be interested in a jax version or no?

### DISCUSSION THREAD

**Discussion by lucidrains** [lucidrains/DALLE2-pytorch#8] [Ideas]
question is in title

**Comment by lucidrains** [comment]
if built, will use jax / [equinox](https://github.com/patrick-kidger/equinox) / and [optax](https://github.com/deepmind/optax)

**Comment by d0rc** [comment]
Yes.

**Comment by lucidrains** [comment]
https://github.com/lucidrains/DALLE2-jax placeholder :man_shrugging:
